tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/BladeCLI.java

```diff
@@ -149,6 +149,36 @@ private static IPath getBladeCLIPath() throws BladeCLIException
         return projectTemplateNames;
     }
 
+    public static String[] getIntegrationPoints() throws BladeCLIException
```

Comment:
Don't add these to BladeCLI.  BladeCLi should just mimic the actually functionalities of the blade cli program.  These two methods should be add in the class that needs the integration points.